crates/kitsune_p2p/kitsune_p2p/src/spawn/actor.rs

```diff
@@ -713,7 +713,7 @@ impl KitsuneP2pActor {
                         }
                     })
                     .await;
-                tracing::warn!("KitsuneP2p tx2:ep poll shutdown");
+                panic!("KitsuneP2p: networking poll shutdown");
```

Comment:
Holochain can no longer communicate with any other node if this so-called "warning" is printed... hence conversion to a panic. The co-implemented update to tx5 -alpha.14 removes all the cases I could discover that would have triggered this.